internal/auth0/branding/resource.go

```diff
@@ -115,18 +123,12 @@ func readBranding(_ context.Context, d *schema.ResourceData, m interface{}) diag
 	result := multierror.Append(
 		d.Set("favicon_url", branding.GetFaviconURL()),
 		d.Set("logo_url", branding.GetLogoURL()),
+		d.Set("colors", flattenBrandingColors(branding.GetColors())),
+		d.Set("font", flattenBrandingFont(branding.GetFont())),
+		d.Set("universal_login", nil),
 	)
-	if _, ok := d.GetOk("colors"); ok {
```

Comment:
This was a very weird custom logic we had within the read func that was only assigning the API data if the config blocks were present.